src/Enum/ActivityAttribute.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Temporal\OpenTelemetry\Enum;
+
+enum ActivityAttribute: string
+{
+    case Id = 'activity.id';
```

Comment:
I don't think we need to prefix all the span attrributes with the type of thing they're being applied to, since that ought to be clear from the span name.

Additionally, something like `activity.workflow_type` and `workflow.type` should really be the same attribute name since they refer to the same thing, so that when I want to filter spans by workflow type, that just works properly. IE: They should probably both be `workflow_type`.

Reply:
Was it resolved with [550409b](https://github.com/temporalio/sdk-php-interceptors-opentelemetry/pull/1/commits/550409bd355375988686dbfcadd2c08cc80a517b)?